src/components/calcite-dropdown/calcite-dropdown.scss

```diff
@@ -1,77 +1,59 @@
 :host {
-  position: relative;
-  display: inline-flex;
-  flex-grow: 1;
+  @apply relative flex-grow;
```

Comment:
@julio8a - missing `inline-flex` here - I think that's why those tests are failing

Reply:
Jack had suggested it wasn't needed, but I guess it was.